tests/WebAppCallsMicrosoftGraph/Pages/Index.cshtml.cs

```diff
@@ -38,7 +38,9 @@ public async Task OnGet()
                 }
                 ViewData["name"] = user.DisplayName;
 
-                var graphData = await _downstreamWebApi.CallWebApiForUserAsync("GraphBeta");
+                var graphData = await _downstreamWebApi.CallWebApiForUserAsync(
```

Comment:
let's comment this out and re-add what we had before

Reply:
Done. Thanks @jennyf19